lib/diagnostic.js

```diff
@@ -18,7 +18,24 @@ const fs = require('fs');
 let inFile = process.argv[2];
 
 let main = (file) => {
-    /* your response here */
+  fs.readFile(file, { encoding: 'utf8' }, (error, content) => {
+    if (error){
+      console.error(error);
+    }
+
+    let lines = content.split('\n');
+    let sum = 0;
+    lines.forEach((line) => {
+      if (isNaN(line)){
+        console.log(line + "isn't a number!");
```

Comment:
Not quite - when a line is not a number, you should throw an error (thereby halting execution), not just print a message in the console.